modules/zipkin-http-exporter/src/main/scala/io/janstenpickle/trace4cats/zipkin/ZipkinSpan.scala

```diff
@@ -0,0 +1,106 @@
+package io.janstenpickle.trace4cats.zipkin
+
+import cats.Foldable
+import cats.syntax.foldable._
+import cats.syntax.show._
+import io.circe.generic.semiauto.deriveEncoder
+import io.circe.syntax._
+import io.circe.{Encoder, Json}
+import io.janstenpickle.trace4cats.`export`.SemanticTags
+import io.janstenpickle.trace4cats.model.AttributeValue.{LongValue, StringValue}
+import io.janstenpickle.trace4cats.model._
+import io.janstenpickle.trace4cats.zipkin.ZipkinSpan.Endpoint
+
+import java.util.concurrent.TimeUnit
+
+// implements https://zipkin.io/zipkin-api/zipkin2-api.yaml
+// does not support 'annotations' and 'debug' fields
+case class ZipkinSpan(
+  traceId: String,
+  name: String,
+  parentId: Option[String],
+  id: String,
+  kind: Option[String],
+  timestamp: Long,
+  duration: Long,
+  shared: Option[Boolean],
+  localEndpoint: Option[Endpoint],
+  remoteEndpoint: Option[Endpoint],
+  tags: Map[String, String]
+)
+
+object ZipkinSpan {
+
+  case class Endpoint(serviceName: Option[String], ipv4: Option[String], ipv6: Option[String], port: Option[Int]) {
+    def nonEmpty: Boolean = serviceName.isDefined || ipv4.isDefined || ipv6.isDefined || port.isDefined
+  }
+
+  private def toTags(span: CompletedSpan): Map[String, String] = {
+    val statusTags = SemanticTags.statusTags(prefix = "", requireMessage = false)(span.status)
+    (span.attributes ++ statusTags).view.mapValues(_.show).toMap
```

Comment:
I feel we don't need the `service.name` tag since it's already in the local endpoint in all cases.